File: java/src/main/java/org/vosk/LogLevel.java
package org.vosk;

public enum LogLevel {
    WARNINGS(-1),  // Print warning and errors
    INFO(0),       // Print info, along with warning and error messages, but no debug
    DEBUG(1);      // Print debug info

    private final int value;

    LogLevel(int value) {
        this.value = value;
    }

    public int getValue() {
        return this.value;
    }
}

File: kotlin/src/commonMain/kotlin/org/vosk/LogLevel.kt
package org.vosk

/**
 * Log level for Kaldi messages.
 *
 * 26 / 12 / 2022
 */
enum class LogLevel(val value: Int) {

	/**
	 * Don't print info messages
	 */
	WARNINGS(-1),

	/**
	 * Default value to print info and error messages but no debug
	 */
	INFO(0),

	/**
	 * More verbose mode
	 */
	DEBUG(1);
}